README.md

```diff
@@ -1183,27 +1184,43 @@ Flow is not supported
 When using models, you write an interface, along with its property types, that will be used to perform type checks at runtime.
 What about compile time? You can use TypeScript interfaces to perform those checks, but that would require writing again all the properties and their actions!
 
-Good news! You don't need to write it twice! Using the `typeof` operator of TypeScript over the `.Type` property of a MST Type will result in a valid TypeScript Type!
+Good news! You don't need to write it twice!
+
+There are three kinds of types available, plus one helper type:
+- `Instance<typeof TYPE>` or `Instance<typeof VARIABLE>` is the node instance type. (Legacy form is `typeof MODEL.Type`).
+- `InSnapshot<typeof TYPE>` or `InSnapshot<typeof VARIABLE>` is the input (creation) snapshot type. (Legacy form is `typeof MODEL.CreationType`).
```

Comment:
Suggestion: `SnapshotIn` and `SnapshotOut`? That makes it slightly easier to discover the relation ship between those two

Reply:
done